wdl-analysis/src/queue.rs

```diff
@@ -735,3 +886,91 @@ pub(crate) fn format_panic_payload(payload: &Box<dyn std::any::Any + Send>) -> S
         "unknown panic payload".to_string()
     }
 }
+
+/// Finds an identifier token at the specified byte offset in the CST.
+fn find_identifier_at_offset(node: &SyntaxNode, offset: usize) -> Result<Option<String>> {
+    let text_size_offset = TextSize::try_from(offset)
+        .map_err(|_| anyhow!("offset {} out of bounds for TextSize", offset))?;
+
+    if let Some(token) = node
+        .token_at_offset(text_size_offset)
+        .find(|t| t.kind() == SyntaxKind::Ident)
+    {
+        let token_range = token.text_range();
+        if token_range.contains_inclusive(text_size_offset)
+            && token_range.start() != token_range.end()
```

Comment:
I think `token_range.start() != token_range.end()` can be `!token_range.is_empty()`.